testsuite/testsuite-jsp/pom.xml

```diff
@@ -49,6 +64,12 @@
             <version>${project.version}</version>
             <executions>
               <execution>
+                <id>package</id>
```

Comment:
Right, it doesnt work without it, because it requires proper jboss-module loading in order replace the JAXP APIs that are also provided by the JDK. The majority of testsuite uses wildfly-swarm:run, which juggles classloading scopes outside modules and here it doesn't work properly.